File: backend/src/main/java/by/fpmibsu/findafriend/controller/commands/users/CreateUserCommand.java
package by.fpmibsu.findafriend.controller.commands.users;

import by.fpmibsu.findafriend.application.mediatr.Request;

public class CreateUserCommand extends Request<Boolean> {
    public String name;
    public String surname;
    public String login;
    public String password;
    public String email;
    public String phoneNumber;
}

File: BackendNET/Backend/Commands/Users/CreateUser/CreateUserCommand.cs
using MediatR;

namespace Backend.Commands.Users.CreateUser;

public class CreateUserCommand : IRequest<bool>
{
    public string Name { get; set; } = "";
    public string Surname { get; set; } = "";
    public string Login { get; set; } = "";
    public string Password { get; set; } = "";
    public string Email { get; set; } = "";
    public string PhoneNumber { get; set; } = "";
}